README.md

```diff
@@ -170,39 +146,6 @@ python3
 >>> elements = partition_text(filename="example-docs/fake-text.txt")
 ```
 
-
-## :coffee: Installation Instructions for Local Development
```

Comment:
This is moved below Quick Tour so that readers can more quickly find information about using the library.